packages/clerk-js/src/ui/elements/ApplicationLogo.tsx

```diff
@@ -49,18 +65,7 @@ export const ApplicationLogo = (props: ApplicationLogoProps) => {
         props.sx,
       ]}
     >
-      <Image
-        ref={imageRef}
-        elementDescriptor={descriptors.logoImage}
-        alt={applicationName}
-        src={imageSrc}
-        size={200}
-        onLoad={() => setLoaded(true)}
-        sx={{
-          display: loaded ? 'inline-block' : 'none',
-          height: '100%',
-        }}
-      />
+      {homeUrl ? <RouterLink to={homeUrl}>{image}</RouterLink> : image}
```

Comment:
❓ Why not just an anchor tag?

Reply:
`RouterLink` renders a `Link` that renders an anchor tag :)